TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/DrivebaseAuto.java

```diff
@@ -0,0 +1,104 @@
+
+package org.firstinspires.ftc.teamcode.opmodes;
+
+import com.acmerobotics.roadrunner.TrajectoryActionBuilder;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import androidx.annotation.NonNull;
+import com.acmerobotics.dashboard.config.Config;
+import com.acmerobotics.dashboard.telemetry.TelemetryPacket;
+import com.acmerobotics.roadrunner.Action;
+import com.acmerobotics.roadrunner.Pose2d;
+import com.acmerobotics.roadrunner.SequentialAction;
+import com.acmerobotics.roadrunner.Vector2d;
+import com.acmerobotics.roadrunner.ftc.Actions;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.DcMotorEx;
+import org.firstinspires.ftc.teamcode.roadrunner.MecanumDrive;
+
+import org.firstinspires.ftc.teamcode.*;
+
+@Autonomous(name="Drive-Base-Auto", group = "Real")
+public class DrivebaseAuto extends LinearOpMode {
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+
+        Robot robot = new Robot(this);
+        robot.configureAutoSetting();
+
+        int INCH_TO_TILE = org.firstinspires.ftc.teamcode.Constants.INCH_TO_TILE;
```

Comment:
Making a local variable of a constant is both a waste of memory and also defeats the purpose of a constant - its immutability.